pgdrive/scene_manager/spawn_manager.py

```diff
@@ -20,8 +23,8 @@ class SpawnManager:
     """
     FORCE_AGENT_NAME = "force_agent_name"
     REGION_DETECT_HEIGHT = 10
-    REGION_DETECT_LONGITUDE = 8
-    REGION_DETECT_LATERAL = 3
+    RESPAWN_REGION_LONGITUDE = 8.
+    RESPAWN_REGION_LATERAL = 3.
```

Comment:
Oh why bother? Does config system raise error if not do so?

Reply:
I did it casually...